cpp/src/backends/process/model_worker_socket.cc

```diff
@@ -34,14 +35,26 @@ int main(int argc, char* argv[]) {
     }
     torchserve::Logger::InitLogger(FLAGS_logger_config_path);
 
+    if (FLAGS_metrics_config_path.empty()) {
+      FLAGS_metrics_config_path =
+          std::string() +
+          std::filesystem::canonical(gflags::ProgramInvocationName())
+              .parent_path()
+              .c_str() +
+          PATH_SEPARATOR + ".." + PATH_SEPARATOR + ".." + PATH_SEPARATOR +
+          ".." + PATH_SEPARATOR + "ts" + PATH_SEPARATOR + "configs" +
+          PATH_SEPARATOR + "metrics.yaml";
+    }
+
```

Comment:
one initMetrics function should be here to load metrics into caching. It is similar as logger which is initialized at line 36.

Reply:
Updated implementation